app/views/search/_result.html.erb

```diff
@@ -1,40 +1,28 @@
-<li>
+<li class="result">
   <h2 class="record-title">
     <span class="sr">Title: </span><%= link_to(result['title'], record_path(result['timdexRecordId'])) %>
   </h2>
 
-  <% result['contentType']&.each do |type| %>
-    <p>
-      <%= type %>
-    <p>
-  <% end %>
+  <p class="pub-info">
+    <span><%= result['contentType']&.each { |type| type['value'] }.join(' ; ') %></span>
+    <span>
+      <% result['dates']&.each do |date| %>
+        <%= date['value'] if date['kind'] == 'Publication date' %>
+      <% end %>
+    </span>
+  </p>
 
   <% contributors = [] %>
   <% result['contributors']&.each do |contributor| %>
-    <% contributors << "#{contributor['value']} (#{contributor['kind']})" %>
+    <% contributors << contributor['value'] %>
```

Comment:
For some articles, the contributors list will literally be hundreds of people (thanks high-energy physics for including everyone in the field on every article!). We can come back to this later, but something like displaying up to 10 with `et al` on the results view seems safe-ish.